### DISCUSSION TITLE
Please add feature select screen area

### DISCUSSION THREAD

**Discussion by sirhvd** [GitMyCode/Terraria-FishingBot#11] [Ideas]
Instead of a fixed screen capture position, you can add the option "Select screen area". It's would better instead of playing in 1920x1080 or 800x600 resolution. 

And one more thing, using Cheat Engine can scan for fishing role bobber animation. But I don't know how to find a static address. 

Anyway, thank you.

**Comment by GitMyCode** [comment]
yeah i could read the memory but it would take me hours to find the address and would propably change with patches anyway. I wanted to have fun with just looking at the screen so yeah it is probably not the best solution but it is what i wanted to do

**Comment by GitMyCode** [comment]
To add the select screen area I thought of it and will maybe do it at some point with the new dotnet MAUI for fun. But again this is not that simple to implement with a UX that is easy to understand. And anyway in terraria you are in the middle to the screen and the bot is used when afk, so there is not much advantage in being able to selelect the area.

You are forced to be in 1920x1080 or lower because of the pixels. When you play higher resolution terraria will change a bit the models by scalling up so then the bot won't be able to detect the hook
